specification/automation/resource-manager/Microsoft.Automation/stable/2018-01-15/dscCompilationJob.json

```diff
@@ -0,0 +1,608 @@
+            "schema": {
+              "$ref": "#/definitions/DscCompilationJob"
+            }
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "DscCompilationJob"
+        ],
+        "operationId": "DscCompilationJob_Get",
+        "description": "Retrieve the Dsc configuration compilation job identified by job id.",
+        "externalDocs": {
+          "url": "http://aka.ms/azureautomationsdk/dsccompilationjoboperations"
+        },
+        "x-ms-examples": {
+          "Get a DSC Compilation job": {
+            "$ref": "./examples/getCompilationJob.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./definitions.json#/parameters/AutomationAccountNameParameter"
+          },
+          {
+            "name": "compilationJobName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The the DSC configuration Id."
+          },
+          {
+            "$ref": "./definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "./definitions.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DscCompilationJob"
+            }
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Automation/automationAccounts/{automationAccountName}/compilationjobs": {
+      "get": {
+        "tags": [
+          "DscCompilationJob"
+        ],
+        "operationId": "DscCompilationJob_ListByAutomationAccount",
+        "description": "Retrieve a list of dsc compilation jobs.",
+        "externalDocs": {
+          "url": "http://aka.ms/azureautomationsdk/compilationjoboperations"
+        },
+        "x-ms-examples": {
+          "List DSC Compilation job in Automation Account": {
+            "$ref": "./examples/listCompilationJobsByAutomationAccount.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./definitions.json#/parameters/AutomationAccountNameParameter"
+          },
+          {
+            "name": "$filter",
```

Comment:
>$filter [](start = 21, length = 7)

x-ms-odata extension would be valuabe for this operation

Reply:
Thanks. I'll talk with the team and take this up as a separate PR as it is not a blocker right now.

Reply:
As this is a comment for a swagger in the stable folder, I strongly recommend to get whatever is inside this folder right in one PR. This way you will avoid breaking changes on generated SDKs.

Reply:
Done.

Reply:
Fixed